packages/cody/src/lib/hooks/ui-files/command-files/app/components/__service__/commands/__className__.tsx__tmpl__

```diff
@@ -9,7 +9,7 @@ interface OwnProps {
 
 }
 
-type <%= className %>Props = OwnProps;
+type <%= className %>Props = OwnProps & WithCommandButtonProps;
```

Comment:
I've modified the UI command react component template so that the component can forward `CommandButtonProps` to the child `CommandButton` component.

This is needed to change the style of the command button when using the command react component. See usage in DataSelectWidget below.